docs/development/dev-process.rst

```diff
@@ -23,7 +23,7 @@ see in particular `Git and GitHub learning resources <https://help.github.com/ar
     $ git clone https://github.com/twisted/twisted twisted
     $ python3 -m venv ./venv
     $ . venv/bin/activate
-    $ pip install -e .[dev]
+    $ pip install -e .[dev,test]
```

Comment:
Thanks. We can have this for now.

In an ideal case, dev will also install test.
But for another PR.

Free free to send a PR if you have time to help with this.